### PR TITLE
docs(website): add v0.0.3 release blogpost

### PR DESCRIPTION
**Pull request by christophenne**
## ✅ Checks
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required

### CODE REVIEW

File: website/blog/2024-02-27-release-package-updates/index.mdx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+have gone into the support for such updates via CLI and GUI. It's as easy as typing `glasskube update` into the command line,
+or a button click in the GUI.
+
+During package installation you now can decide whether you want automatic updates for the package, in which case the
+Glasskube package operator will take care of keeping you up to date at all times. You can of course opt out of this feature
+if you prefer to handle updates manually.
+
+![Package Updates](./package-updates.png)
+
+For technical details, please have a look at the [package operator documentation](/docs/components/package-operator/#handling-package-updates).
```
Comment (pmig) [inline website/blog/2024-02-27-release-package-updates/index.mdx:47]: I would love to also see a short section about installing a specific version with the new `--version` flag

File: website/blog/2024-02-27-release-package-updates/index.mdx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+* The `glasskube describe` command – and its GUI-complement, the package detail page – have been implemented for you to have a more detailed
+view on each of the packages available in Glasskube.
+* `glasskube list` has been extended with the new flags `--outdated` – listing only outdated packages, and `--show-latest` – showing the
+latest available version for each package.
+* All commands check for a newer version of Glasskube, and notify you if you are not up to date anymore.
+* Glasskube is now even easier to set up: The GUI comes with handy pages for selecting a kubeconfig and for bootstrapping Glasskube in a cluster.
+All CLI commands will also support you in setting up Glasskube in your cluster.
+* Streaming any kind of package status change directly into the GUI, making it easier for you to recognize when something is happening in the background.
+
```
Comment (pmig) [inline website/blog/2024-02-27-release-package-updates/index.mdx:61]: The information that glasskube is outdated and needs to get bootstrapped again would be also a nice mention

**Review by pmig — CHANGES_REQUESTED** [review]
Glasskube v0.0.3 is getting real. I think that the two additions will make the blog post even better.

File: website/blog/2024-02-27-release-package-updates/index.mdx (hunk 2 above)
Comment (christophenne) [reply website/blog/2024-02-27-release-package-updates/index.mdx:61]: It's mentioned below in "How to upgrade". Or if you mean the outdated-detection, this is what I tried to describe in the third bullet point.

**Comment by christophenne** [comment]
package-updates

**Comment by christophenne** [comment]
package-updates

**Comment by pmig** [comment]
I quickly added some drop shadow
 
screenshotr_2024-2-27T17-9-7

**Review by pmig — APPROVED** [review]

**Pull request merged**